srcpkgs/fractal/template

```diff
@@ -8,7 +8,7 @@ hostmakedepends="cargo pkg-config glib-devel gettext"
 makedepends="gtk+3-devel libglib-devel openssl-devel gstreamer1-devel
  cairo-devel gettext-devel gst-plugins-base1-devel gst-plugins-bad1-devel
  gst1-editing-services-devel gtksourceview4-devel pango-devel gspell-devel
- gmp-devel libsecret-devel libhandy-devel rust-std"
+ gmp-devel libsecret-devel libhandy-devel rust-std gst1-editing-services-devel python3-devel cmake"
```

Comment:
Why is `cmake` needed? It builds fine without it for me.

Reply:
A bunch of packages needed these... that's weird, that was the case for bijiben as well. Welp imma remove this cmake.

Reply:
there, it's now gone.

Reply:
What does that mean, "needed these"?

Projects with meson.build don't even use cmake at all unless the project does dependency('some-cmake-project'), in which case meson executes a small cmake dummy project and parses the cmake debug trace to see what cmake detected via a *-config.cmake; this is an alternative to pkg-config.

Meson will almost always print that it is checking for cmake as a system command, if it doesn't find an optional dependency. The fact that it prints "NO" doesn't mean that cmake is a dependency and should be installed as makedepends... it is just an informational message that cmake is not installed.

The actual error, if an error occurs, is always "some-package could not be found, tried pkg-config and cmake" and the solution is to install the actual dependency, not to try installing cmake.

Reply:
(note that installing cmake won't even help, in most cases.)

Reply:
> What does that mean, "needed these"?
> 
> Projects with meson.build don't even use cmake at all unless the project does dependency('some-cmake-project'), in which case meson executes a small cmake dummy project and parses the cmake debug trace to see what cmake detected via a *-config.cmake; this is an alternative to pkg-config.
> 
> Meson will almost always print that it is checking for cmake as a system command, if it doesn't find an optional dependency. The fact that it prints "NO" doesn't mean that cmake is a dependency and should be installed as makedepends... it is just an informational message that cmake is not installed.
> 
> The actual error, if an error occurs, is always "some-package could not be found, tried pkg-config and cmake" and the solution is to install the actual dependency, not to try installing cmake.

when i ran the testings at some point the checks returned an error with cmake labeled as missing... But it doesn't happen  anymore. I'm not actually sure of what's going on.

But if it works without, well it's good for me.